Categories+Tags

cpastre1 opened the pull request:
This includes styles for categorizing and tagging content, examples of where both tags and categories would be displayed on content pages, and landing page templates for when tags and categories are clicked. 

Also includes byline formatting, extended horizontal rules, and content containers (used for featured category content).

cpastre1 committed:
State Widget Styling

cpastre1 committed:
Began building button+form element

Got button element to attach to form element with button-form class.
Added hover-over highlight to feature buttons. Button+form element is
still not spanning columns/divs properly

cpastre1 committed:
Fixed Email Widget

Replaced button/form on email widget

cpastre1 committed:
Tag Styles

Added tags stylesheet. Built out tag styles.

cpastre1 committed:
Tag Color

Changed tags to be tf-blue on hover.

cpastre1 committed:
Category Styles

Finished styling category labels. Built sample tagged/categorized
content section. Adjusted link styles in tags and categories.

cpastre1 committed:
Fixed links in categories and tags

cpastre1 committed:
Bylines

Began building date and byline section.

cpastre1 committed:
Added category page template

cpastre1 committed:
Renamed Category Page

cpastre1 committed:
Adjusted tag + category styles

cpastre1 committed:
Added byline section

cpastre1 committed:
Reformatted tags + categories

cpastre1 committed:
Finished Byline Styles

cpastre1 committed:
Rearranged category page header elements

cpastre1 committed:
Changed name of categories and tags file

cpastre1 committed:
Styled header section of category page template

cpastre1 committed:
Content Cards

Built content cards and added featured section to category pages

cpastre1 committed:
Added hover effect to content cards

cpastre1 committed:
Added pagination to categories template

Pagination is not fully built out yet and is serving as a placeholder at
this point.

cpastre1 committed:
Built tag page template

Added landing page for tags. Included related tags section with list of
links.

tvanantwerp commented:
The tallest feature block in a row during hover grows the row height by 4px due to bottom border addition. Bottom border also clipped in top left and right edges due to the regular left and right border. Suspect fixing this bit will require some creativity.

![image](https://cloud.githubusercontent.com/assets/2487968/16012789/eed6f118-3158-11e6-936e-f7b927a383a4.png)

tvanantwerp commented:
![image](https://cloud.githubusercontent.com/assets/2487968/16013845/0486f090-315d-11e6-951e-1368fe2cf531.png)

Images within cards have bottom radii, not sure if it's intentional. Also wondering if we want the gray border for the card to encompass the image at all--might only want to have the gray border wherever there isn't an image, such as for the text `.content`.

**Nevermind** looks like that's something I did while fiddling with the code.

tvanantwerp committed:
Change 0em and none to 0

cpastre1 committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into Categories+Tags

# Conflicts:
#	dist/index.html

cpastre1 committed:
Merge branch 'Categories+Tags' of https://github.com/TaxFoundation/tf-web-styles into Categories+Tags

tvanantwerp committed:
Refactor card

tvanantwerp committed:
Refactor rounded card corners

tvanantwerp committed:
Refactor card images

tvanantwerp committed:
Refactor card content

tvanantwerp committed:
Apply date styles directly to p

tvanantwerp committed:
Fix HTML errors in index

tvanantwerp committed:
Refactor bylines

tvanantwerp committed:
Correct article listing in category-page

tvanantwerp committed:
Begin refactoring categories-tags

tvanantwerp committed:
Categories refactored, styles may need adjustment

tvanantwerp committed:
Tags refactored

tvanantwerp committed:
Tweak to tag hover styles

tvanantwerp committed:
Merge remote-tracking branch 'refs/remotes/origin/Categories+Tags' into category-tag-refactor

cpastre1 committed:
Fixed issues highlight in PR

Addressed content card border issues. Fixed featured content sections on
category and main style page so that they so not cause overflow issues.

cpastre1 committed:
Merge remote-tracking branch 'refs/remotes/origin/Categories+Tags' into category-tag-refactor

# Conflicts:
#	src/scss/_content-card.scss
#	src/scss/categories-tags.scss

cpastre1 committed:
Merge pull request #21 from TaxFoundation/category-tag-refactor

Category tag refactor
